lib/Onyx.js

```diff
@@ -1554,21 +1554,7 @@ function setMemoryOnlyKeys(keyList) {
  *     },
  * });
  */
-function init({
-    keys = {},
-    initialKeyStates = {},
-    safeEvictionKeys = [],
-    maxCachedKeysCount = 1000,
-    captureMetrics = false,
-    shouldSyncMultipleInstances = Boolean(global.localStorage),
-    debugSetState = false,
-} = {}) {
-    if (captureMetrics) {
-        // The code here is only bundled and applied when the captureMetrics is set
-        // eslint-disable-next-line no-use-before-define
-        applyDecorators();
-    }
-
+function init({keys = {}, initialKeyStates = {}, safeEvictionKeys = [], maxCachedKeysCount = 1000, shouldSyncMultipleInstances = Boolean(global.localStorage), debugSetState = false} = {}) {
```

Comment:
nab: shouldn't it be formatted like earlier? I think it was more readable

Reply:
That's how prettier formatted it after removing one parameter 😅